firefox-ios/Storage/Rust/RustLogins.swift

```diff
@@ -688,6 +720,23 @@ public class RustLogins: LoginsProtocol {
         }
     }
 
+    public func listLogins(completionHandler: @escaping (Result<[EncryptedLogin], Error>) -> Void) {
```

Comment:
Doesn't this one duplicate the function you added inhttps://github.com/mozilla-mobile/firefox-ios/pull/19206?

Reply:
yes it looks like duplicate stuff, probably the merge order would help here

Reply:
Mh. I tried fixing the conflicts in the Github page directly. Seems this is still showing up but it shouldn't as the other PR was merged 🤔 Can you fix this @imsanchit?

Reply:
@lmarceau please check if this PR can be merged now.

Reply:
@imsanchit It seems that there is a another similar function from the other PR. Should we remove that one to and replace it with this one? 

Line: 765

![image](https://github.com/mozilla-mobile/firefox-ios/assets/6743397/ac1b8be3-3436-4268-b69f-80b0ea11cf0f)